config/routes.rb

```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
   root to: 'application#index'
 
+  require 'sidekiq'
   require 'sidekiq/web'
+  
+  Sidekiq::Web.use Rack::Auth::Basic do |username, password|
```

Comment:
Although in the wiki of Sidekiq this configuration is in `routes`, I prefer doing this in a `sidekiq.rb` initializer. Also I will set this pass only in production.

Reply:
So i have to comment it or leave it uncommented? because @Dinuuu  asked why this was commented if we want to restrict access always